### PR TITLE
Minor update to usage/help text

### PR DESCRIPTION
**Pull request by duglin**
Seemed a little out of date

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by duglin** [comment]
I don't think the test failure is related to my 'usage' text change.

File: main.go [hunk 1]
```diff
@@ -11,18 +11,22 @@ const (
 	version = "0.2"
 	usage   = `Open Container Project runtime
 
-runc is a command line client for running applications packaged according to the Open Container Format (OCF) and is
-a compliant implementation of the Open Container Project specification.
+runc is a command line client for running applications packaged according to
+the Open Container Format (OCF) and is a compliant implementation of the
+Open Container Project specification.
```
Comment (mrunalp) [inline main.go]: It is called Open Container Initiative now.

Comment (duglin) [inline main.go]: think I got it!  thanks!

-Doug

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**